src/h5web/providers/jupyter/jupyter-dev-api.ts

```diff
@@ -32,7 +33,15 @@ export class JupyterDevApi extends JupyterStableApi {
   }
 
   public async getValue(params: ValueRequestParams): Promise<unknown> {
-    return this.fetchData(params);
+    const { path, selection } = params;
+    const [value, entity] = await Promise.all([
+      this.fetchData(params),
+      this.getEntity(path),
+    ]);
+
+    assertDataset(entity);
+
+    return this.flattenValue(value, entity, selection);
```

Comment:
As I need to know the dataset dimensions to flatten, I need to fetch the entity along with the data...

This was not an issue for `JupyterStable` and `Hsds` as the entity was already fetched for other reasons (complex type parsing in Jupyter and dataset assertion in Hsds)

This is not ideal but won't be a problem anymore if we move the flattening in the backend